frontend/package.json

```diff
@@ -17,24 +17,29 @@
     "@bufbuild/protoc-gen-es": "^1.10.0",
     "@trustwallet/wallet-core": "^4.1.0",
     "@types/react-router-dom": "^5.3.3",
+    "7z-wasm": "^1.1.0",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
     "i18next": "^23.12.2",
     "i18next-browser-languagedetector": "^8.0.0",
     "i18next-xhr-backend": "^3.2.2",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
+    "lottie-react": "^2.4.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-i18next": "^15.0.0",
+    "react-qr-code": "^2.0.15",
     "react-router-dom": "^6.25.1",
     "stream-browserify": "^3.0.0",
+    "uuid": "^10.0.0",
     "vite-plugin-node-polyfills": "^0.22.0",
     "vite-plugin-node-stdlib-browser": "^0.2.1"
   },
   "devDependencies": {
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
-    "@typescript-eslint/eslint-plugin": "^8.0.1",
-    "@typescript-eslint/parser": "^8.0.1",
+    "@types/uuid": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^7.18.0",
```

Comment:
why downgrade versions? Needs to keep it and follow current lint configs.

Reply:
Sorry, missed that, going to fix it

Reply:
@0xkobe 
Fixed